README.md

```
@@ -1,31 +1,24 @@
-| Mashrief Bin Zulfiquer       | Mentor       | [mashrief](https://github.com/mashrief)           |
+| Name                   | Role        | GitHub Username                                     |
+| ---------------------- | ----------- | --------------------------------------------------- |
+| Md Emran Hossen        | Team Leader | [Md-EmranHossen](https://github.com/Md-EmranHossen) |
+| Md Rifatul             | Member      | [md-rifatul](https://github.com/md-rifatul)         |
+| FI Pranto              | Member      | [FI-Pranto](https://github.com/FI-Pranto)           |
+| Mashrief Bin Zulfiquer | Mentor      | [mashrief](https://github.com/mashrief)             |
 
+# 🧑‍💻Learning & Project Planning
 
-# 🧑‍💻Learning & Project Planning 
 ### Stack Learning
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)